app/config/custom-environment-variables.json

```diff
@@ -7,6 +7,9 @@
     "oidc": {
       "authority": "FRONTEND_OIDC_AUTHORITY",
       "clientId": "FRONTEND_OIDC_CLIENTID"
+    },
+    "notifications": {
+      "homeBanner": "FRONTEND_NOTIFICATION_HOME_BANNER"
```

Comment:
Please have this be inserted in alphabetical order. In this case, it should come before the OIDC keys.

Reply:
Fixed